pb/bitcoin/service.proto

```diff
@@ -195,3 +199,17 @@ message Output {
   // Amount of coins to be sent
   string value = 2;
 }
+
+message GetKeypairRequest {
+  // Optional seed
+  string seed = 1;
+  // Network used (Main, Test, Reg...)
+  BitcoinNetwork network = 2;
+}
+
+message GetKeypairResponse {
+  // Extended public key
+  string public_key = 1;
```

Comment:
Let's rename this to `extended_public_key`  or just `xpub`, since it's more than just a "public key".